bevy_rapier2d/Cargo.toml

```diff
@@ -46,7 +46,7 @@ wasm-bindgen = ["rapier2d/wasm-bindgen"]
 serde-serialize = ["rapier2d/serde-serialize", "bevy/serialize", "serde"]
 enhanced-determinism = ["rapier2d/enhanced-determinism"]
 headless = []
-async-collider = ["bevy/bevy_asset", "bevy/bevy_scene"]
+async-collider = ["bevy/bevy_asset", "bevy/bevy_scene", "bevy/bevy_render"]
```

Comment:
that's not relevant to debug-render-3d, but a run through https://github.com/frewsxcv/cargo-all-features revealed `cargo check --no-default-features --features async-collider,dim3` suffered from a failed compilation too.